app/Http/Livewire/View/CalendarlistComponent.php

```diff
@@ -45,11 +45,13 @@ class CalendarlistComponent extends Component
     public $closeModal = false;
     public $callbackModal = false;
 
+    public $format = 'd.m.Y';
+    public $format_calendar = '0';
+
     public function render()
     {
         $user = Auth::user();
-        $this->now = date_format(new DateTime('now'),"Y-m-d");
-        //dd($user);
+        $this->now = date_format(new DateTime('now'),"d-m-Y");
```

Comment:
Different from Y-m-d

Reply:
<img width="573" alt="image" src="https://user-images.githubusercontent.com/98778582/159068546-4173208c-2fd3-4da7-adac-b1405d675dca.png">

That's right, that's how it should be. The data is used only for data sorting